### ISSUE TITLE
Failing during `yarn start`

### ISSUE THREAD

**Issue by BigBlueHat** [apache/incubator-annotator#28]
The tests reference a `../lib` directory which is absent (perhaps auto-generated at runtime or something?) and it causes the following errors:
```
ERROR in ./packages/selector/test/index.mjs
Module not found: Error: Can't resolve '../lib' in 'C:\Users\byoung2\dev\annotator\annotator\p
ackages\selector\test'
 @ ./packages/selector/test/index.mjs 1:0-35 4:21-29
 @ ./node_modules/multi-entry-loader?include=./packages/*/test/**/*.mjs
 @ ./node_modules/mocha-loader!./node_modules/multi-entry-loader?include=./packages/*/test/**/
*.mjs
 @ multi chai/register-assert mocha-loader!multi-entry-loader?include=./packages/*/test/**/*.m
js

ERROR in ./packages/range/test/cartesian.mjs
Module not found: Error: Can't resolve '../lib/cartesian.mjs' in 'C:\Users\byoung2\dev\annotat
or\annotator\packages\range\test'
 @ ./packages/range/test/cartesian.mjs 5:0-47 48:19-26
 @ ./node_modules/multi-entry-loader?include=./packages/*/test/**/*.mjs
 @ ./node_modules/mocha-loader!./node_modules/multi-entry-loader?include=./packages/*/test/**/
*.mjs
 @ multi chai/register-assert mocha-loader!multi-entry-loader?include=./packages/*/test/**/*.m
js
i ｢wdm｣: Failed to compile.
```

Full log (containing the above) is below:
```bash
$ yarn start
yarn run v1.5.1
$ webpack-serve
i ｢hot｣: webpack: Compiling...
i ｢hot｣: WebSocket Server Listening at localhost:8081
i ｢serve｣: Project is running at http://localhost:8080
i ｢serve｣: Server URI copied to clipboard
i ｢hot｣: webpack: Compiling Done
× ｢wdm｣: Hash: 48894b03e48351b3f485
Version: webpack 4.1.1
Time: 13734ms
Built at: 2018-3-28 14:25:09
  Asset      Size  Chunks                    Chunk Names
demo.js   949 KiB    demo  [emitted]  [big]  demo
test.js  2.48 MiB    test  [emitted]  [big]  test
Entrypoint demo [big] = demo.js
Entrypoint test [big] = test.js
[./demo/index.js] 5.17 KiB {demo} [built]
[./demo/mark.js] 804 bytes {demo} [built]
[./node_modules/@babel/runtime/helpers/builtin/es6/asyncToGenerator.js] 709 bytes {test} {demo
} [built]
[./node_modules/chai/index.js] 40 bytes {test} [built]
[./node_modules/chai/register-assert.js] 38 bytes {test} [built]
[./node_modules/mocha-loader/index.js!./node_modules/multi-entry-loader/index.js?include=./pac
kages/*/test/**/*.mjs!./] ./node_modules/mocha-loader!./node_modules/multi-entry-loader?includ
e=./packages/*/test/**/*.mjs 776 bytes {test} [built]
[./node_modules/webpack-hot-client/client/index.js?69a6d200-876a-4461-8380-b5c66ef55c63] (webp
ack)-hot-client/client?69a6d200-876a-4461-8380-b5c66ef55c63 1.78 KiB {test} {demo} [built]
[./node_modules/webpack-hot-client/client/socket.js] (webpack)-hot-client/client/socket.js 1.3
7 KiB {test} {demo} [built]
[./node_modules/webpack/buildin/global.js] (webpack)/buildin/global.js 509 bytes {test} {demo}
 [built]
[./packages/dom/src/index.mjs] 1.74 KiB {demo} [built]
[./packages/fragment-identifier/src/index.mjs] 1.31 KiB {demo} [built]
   [0] multi ./demo/index.js 28 bytes {demo} [built]
   [1] multi chai/register-assert mocha-loader!multi-entry-loader?include=./packages/*/test/**
/*.mjs 40 bytes {test} [built]
   [2] multi webpack-hot-client/client?69a6d200-876a-4461-8380-b5c66ef55c63 ./demo/index.js 40
 bytes {demo} [built]
   [3] multi webpack-hot-client/client?69a6d200-876a-4461-8380-b5c66ef55c63 chai/register-asse
rt mocha-loader!multi-entry-loader?include=./packages/*/test/**/*.mjs 52 bytes {test} [built]
    + 209 hidden modules

ERROR in ./packages/selector/test/index.mjs
Module not found: Error: Can't resolve '../lib' in 'C:\Users\byoung2\dev\annotator\annotator\p
ackages\selector\test'
 @ ./packages/selector/test/index.mjs 1:0-35 4:21-29
 @ ./node_modules/multi-entry-loader?include=./packages/*/test/**/*.mjs
 @ ./node_modules/mocha-loader!./node_modules/multi-entry-loader?include=./packages/*/test/**/
*.mjs
 @ multi chai/register-assert mocha-loader!multi-entry-loader?include=./packages/*/test/**/*.m
js

ERROR in ./packages/range/test/cartesian.mjs
Module not found: Error: Can't resolve '../lib/cartesian.mjs' in 'C:\Users\byoung2\dev\annotat
or\annotator\packages\range\test'
 @ ./packages/range/test/cartesian.mjs 5:0-47 48:19-26
 @ ./node_modules/multi-entry-loader?include=./packages/*/test/**/*.mjs
 @ ./node_modules/mocha-loader!./node_modules/multi-entry-loader?include=./packages/*/test/**/
*.mjs
 @ multi chai/register-assert mocha-loader!multi-entry-loader?include=./packages/*/test/**/*.m
js
i ｢wdm｣: Failed to compile.
```

I changed the `../lib` references to `../src` in that case...but it happened again in another test. I'm not sure (yet) if this was a path change that got missed in the code, or if there's something not running at runtime that should be run. 🏃‍

**Comment by tilgovi** [comment]
I just noticed this, too. I've got some WIP to address it. I'll assign myself.

For the moment, you can always run build and then test and they should pass.

**Comment by tilgovi** [comment]
Although, that won't work in the webpack tests, I think.

**Comment by tilgovi** [comment]
Closed in c1f7fa1dfd84c27537cc20d244cfd48eb65462cd
